common_sensor_launch/launch/velodyne_VLP16.launch.xml

```diff
@@ -14,6 +14,8 @@
   <arg name="view_direction" default="0.0"/>
   <arg name="view_width" default="6.28"/>
   <arg name="vehicle_mirror_param_file"/>
+  <arg name="use_pointcloud_container" default="False"/>
```

Comment:
It should be okay with capitalized "False", but it maybe better if it is aligned with other "true"/"false"

```suggestion
  <arg name="use_pointcloud_container" default="false"/>
```